packages/composables/src/utils/cookies.ts

```diff
@@ -0,0 +1,5 @@
+export const getCurrencyCookie = (context: any) => {
```

Comment:
I'm just thinking - we have this thing in cart, that there's a middleware in `api-client` that fetches a cookie and sets it inside request's config (see https://github.com/vuestorefront/spree/blob/d11d700bb4a1900c35eff0141242ca5aba912d39/packages/api-client/src/api/authentication/integration.ts#L6).

Then inside any api-client method, we're able to fetch it like this: https://github.com/vuestorefront/spree/blob/d11d700bb4a1900c35eff0141242ca5aba912d39/packages/api-client/src/api/authentication/getCurrentBearerOrCartToken.ts#L11

Would it make sense to use the same for handling currencies (e.g. when you call getProduct, the api-client is aware of currency based on cookie and already uses it)? This could simplify part of the workflow, since your UI won't have to fetch currency when calling api-client methods. Let's discuss that ;)